.omegakg_temp/capture_server_logs.md

```
@@ -0,0 +1,212 @@
+2025-12-01 21:08:57,410 - omega_kg.domain.common.embedding_service - INFO - ✓ Embedding Service initialized with provider: ollama, Ollama base URL: http://localhost:11434
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)